news/_posts/core-weekly/2019-03-13-coreweekly-week-10-2019.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 10 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-03-13 14:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 04th of February to Sunday 10th of March 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+An important step for the PrestaShop open source project has been completed last week and it is available today: the nighlty builds and tests dashboard. This is a new website: https://nightly.prestashop.com/.
+
+Each nighht, a new build is created with the newly merged code. Then, the automatic tests are running for this build, and a report is created. It will help contributors to improve the quality of PrestaShop.
```

Comment:
```suggestion
Each night, a new build is created with the newly merged code. Then, the automatic tests are running for this build, and a report is created. It will help contributors to improve the quality of PrestaShop.
```